### PR TITLE
implement a racks command for listing racks

### PR DESCRIPTION
**Pull request by awsmsrc**
## Release Playbook
- [x] Rebase against master
- [x] Pass checks
- [x] Release branch ()
- [ ] Pass CI ()
- [x] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update dev and testing racks
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by anthonyrisinger** [comment]
I like this ticket but I expected it to either list the racks from `~/.convox/auth` and/or call `DescribeStacks` and gather up the `Tags['Rack']` from the CF resources. The former would produce a list of racks known to the client and the latter would produce a region-bound list of racks known to the server.

File: api/controllers/routes.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ func NewRouter() (router *mux.Router) {
 	router.HandleFunc("/instances", api("instances.get", InstancesList)).Methods("GET")
 	router.HandleFunc("/instances/{id}", api("instance.delete", InstanceTerminate)).Methods("DELETE")
 	router.HandleFunc("/instances/keyroll", api("instances.keyroll", InstancesKeyroll)).Methods("POST")
+	router.HandleFunc("/racks", api("rack.list", RegistryList)).Methods("GET")
```
Comment (nzoschke) [inline api/controllers/routes.go]: This is hooked up to the wrong handler

**Comment by nzoschke** [comment]
Interesting expectations @anthonyrisinger.

There's another possibility which is all the racks registered in your Convox Console™, which is what @awsmsrc is laying the groundwork for.

**Comment by anthonyrisinger** [comment]
Ah it looked just hard coded as a placeholder or something for now. "Expected" was only meant to mean "surprising" :-)

(whynotboth)? Dumping from `.convox/auth` would be pretty useful for us, if not connected to Console. I run `jq . ~/.convox/auth` so much I have an alias for it.

File: client/racks.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package client
+
+type Organization struct {
+	Id   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type Rack struct {
```
Comment (nzoschke) [inline client/racks.go]: We also have a struct that covers this in client.System. But I think that one is poorly named, so re-introducing a Rack struct is good.

**Comment by nzoschke** [comment]
:+1:

**Comment by nzoschke** [comment]
I understand your use case around reading .auth, and see how that could make more sense than this endpoint if you're not using our console @anthonyrisinger.

However we decided to go with the remote API that returns all the racks for now, and can re-visit any time later.

**Pull request merged**